genesis/process/argGenesisBlockCreator.go

```diff
@@ -67,4 +67,5 @@ type ArgsGenesisBlockCreator struct {
 	GenesisString    string
 	// created components
 	importHandler update.ImportHandler
+	ChainRunType  common.ChainRunType
```

Comment:
move this on L65 along with the "provided arguments"?

Reply:
Done